### ISSUE TITLE
Problems with building for VMware

### ISSUE THREAD

**Issue by jradcliffe02** [rapid7/metasploitable3#261]
## Issue Description

Having quite a bit of difficulty getting this to run under VMware Workstation 14. Running it the standard build script I get as far as the script trying to startup the virtual machine. The VM never gets started and the script times out and dies. No errors are reported to the terminal when the startup attempts to occur.  

On the same system the VirtualBox install works 100% fine. We're a VMware shop though and I need to get the VMware version to work. 

Thanks in advance! 

## Host System
* OS: Ubuntu 16.10 LTS
* Packer Version: 
* Vagrant Version: 1.9.1
* VirtualBox Version:
* VMware Version: Workstation 14 
This is on a freshly built system.

**Comment by agmic** [comment]
The packer template for ubuntu has a bug in it, it uses the wrong value/syntax for guest_os_type.
file: packer/templates/ubuntu_1404.json
Change line 30
```
--- "guest_os_type": "Ubuntu_64",
+++ "guest_os_type": "ubuntu-64",
```
The same bug is in the pro template.

**Comment by jmartin-r7** [comment]
This is fix by commit 57dad47
